styles.css

```diff
@@ -0,0 +1,338 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+@media (max-width: 75em) {
+  html {
+    font-size: 56.25%;
+  }
+}
+@media (max-width: 65em) {
+  html {
+    font-size: 50%;
+  }
+}
+body {
+  font-family: "Rubik", sans-serif;
+  line-height: 1;
+  font-weight: 400;
+  color: #333;
+  overflow-x: hidden;
+}
+.header {
+  height: 50rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.header::before {
+  content: "";
+  display: block;
+  position: absolute;
+  background: url(https://clickup.com/images/download/bg_gradient__download.svg)
+    no-repeat top center/auto;
+  width: 100%;
+  height: 100%;
+  top: -10%;
+  left: 0;
+  z-index: -2;
+}
+@media screen and (max-width: 44em) {
+  .header {
+    height: 40rem;
+  }
+  .header::before {
+    top: -20%;
+  }
+}
+.header-wrapper {
+  text-align: center;
+}
+.heading-primary {
+  margin-top: 3.2rem;
+  color: #fff;
+  font-size: 5rem;
+}
+@media (max-width: 37em) {
+  .heading-primary {
+    font-size: 4rem;
+  }
+}
+.heading-description {
+  margin-top: 3.5rem;
+  color: #fff;
+  font-size: 1.6rem;
+}
+.middle-section {
+  padding: 2.5rem 2.5rem;
+  overflow: hidden;
+}
+
+.event-select-container {
+  max-width: 120rem;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+}
+
+.subheading {
```

Comment:
I think you should change color of subheading, because when you are checking responsiveness color hides on the background color.

Reply:
I've removed the header part and now subheading should not hiding under it